frontend/src/components/Navigation/Navbar/Navbar.tsx

```
@@ -1,119 +1,139 @@
-      setIsEditing(false);
-    },
-    [onNameChange],
-  );
+              <div className="grid grid-cols-2 gap-4 py-4">
+                <Button
+                  onClick={() => {}}
+                  disabled={false}
+                  className="flex h-32 flex-col items-center justify-center gap-2 p-0"
+                  variant="outline"
+                >
+                  {false ? (
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)